client/src/Components/Toggle.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+const Toggle = (props) => {
+    return (
+        <div className="togglle-btn">
```

Comment:
Is there another CCS selector named `toggle-btn` or is this a typo?

Reply:
probably spelling mistake from my side